cmd/rekor_monitor/main.go

```diff
@@ -56,15 +56,8 @@ func main() {
 	userAgentString := flag.String("user-agent", "", "details to include in the user agent string")
 	flag.Parse()
 
-	if *configFilePath == "" && *configYamlInput == "" {
```

Comment:
I think something got dropped, I don't see logInfoFileName set as a default either.

Reply:
sorry, mixed up with a future PR that I believe takes out logInfoFile from the config and resets it to a default argument- implementing that change in this PR as well

Reply:
reimplementing this as a default flag